server/container_list.go

```diff
@@ -85,6 +85,10 @@ func (s *Server) ListContainers(ctx context.Context, req *pb.ListContainersReque
 	}
 
 	for _, ctr := range ctrList {
+		// Skip over containers that are still being created
+		if !ctr.Created() {
```

Comment:
I'm being silly here, should we need a lock here as create and list are concurrent? Ignore this comment otherwise... We probably don't care as the container won't be added to the results but still.

Reply:
I think that is fine. container will get picked up in next round.